sdk/storage/azdatalake/internal/generated/autorest.md

```diff
@@ -7,7 +7,7 @@ go: true
 clear-output-folder: false
 version: "^3.0.0"
 license-header: MICROSOFT_MIT_NO_VERSION
-input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2020-10-02/DataLakeStorage.json"
+input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json"
```

Comment:
This dependency should be taken from the latest commit instead of directly using the main branch.

Reply:
Makes sense - Changing to commit https://github.com/Azure/azure-rest-api-specs/commit/3f3b51edf8fd0eb65004df390d6ee98e0e23c53d